### PR TITLE
grafana issue

### PR DESCRIPTION
**Pull request by ashiscs**
Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

**Comment by leecalcote** [comment]
@ashiscs if you will include a more descriptive issue title, this will help in understanding what the PR is about.

File: ui/components/GrafanaComponent.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ class GrafanaComponent extends Component {
         }
         if (name === 'grafanaBoardSearch') {
             if (this.boardSearchTimeout) clearTimeout(this.boardSearchTimeout);
-            this.boardSearchTimeout = setTimeout(this.getGrafanaBoards, 500); // to delay the search by a few.
+            this.boardSearchTimeout = setTimeout(this.getGrafanaBoards, 1000); // to delay the search by a few.
```
Comment (leecalcote) [inline ui/components/GrafanaComponent.js]: thanks, @ashiscs. Do you know if 1000ms is generally long enough of a timeout period? Will you be creating another PR to incorporate display of the waiting indicator?

@girishranganathan what design pattern do we want to use for timeouts? Should we sidecar an envoy to Meshery's deployment and use Envoy to control this resiliency logic?

Comment (ashiscs) [reply ui/components/GrafanaComponent.js]: Ok sir!

**Comment by leecalcote** [comment]
@ashiscs also you'll want to check on this item - https://github.com/layer5io/meshery/pull/322#pullrequestreview-297816789

**Comment by leecalcote** [comment]
Uh-oh. We may need to sync this the latest changes from master.

**Comment by leecalcote** [comment]
Build checks passing. I believe this is ready for merge. Is that right, @girishranganathan?

**Comment by girishranganathan** [comment]
@leecalcote it mostly is, but please give me sometime as I am seeing some rare unmarshalling failures in the grafana SDK.
